BricksAndHearts.UnitTests/ControllerTests/ControllerTestsBase.cs

```diff
@@ -33,4 +33,26 @@ protected BricksAndHeartsUser CreateUnregisteredUserInController(Controller _und
         };
         return unregisteredUser;
     }
+    
+    protected BricksAndHeartsUser CreateAdminUserInController(Controller _underTest)
+    {
+        // Arrange
+        var userDbModel = new UserDbModel()
+        {
+            Id = 1,
+            GoogleUserName = "John Doe",
+            GoogleEmail = "test.email@gmail.com",
+            IsAdmin = true,
+            LandlordId = null,
+        };
+
+        var adminUser = new BricksAndHeartsUser(userDbModel, new List<Claim>(), "google");
+        var adminUserPrincipal = new ClaimsPrincipal(adminUser);
+
+        _underTest.ControllerContext = new ControllerContext
+        {
+            HttpContext = new DefaultHttpContext { User = adminUserPrincipal }
+        };
+        return adminUser;
+    }
```

Comment:
I think this whole method can be reduced to
```
var adminUser = CreateAdminUser()
MakeUserPrincipalInController(adminUser, _underTest)
```
as on `main` it looks like these methods already exist in `ControllerTestsBase`. When was the last time you rebased?